gepetuto/lint.py

```diff
@@ -8,21 +8,27 @@
 LOG = logging.getLogger("gepetuto.lint")
 
 
-def lint(tp_id: List[int], **kwargs):
+def lint(tp_id: List[int], file: List[str], **kwargs):
     """Lint python scripts."""
+    file = [Path(f) for f in file]
     LOG.info("linting tutorial sources.")
     for n in tp_id:
         LOG.debug(f"Looking for tp {n}")
-        lint_tp_folder(n)
+        folder = Path(f"tp{n}")
+        for python_file in folder.glob("*.py"):
+            if file == []:
+                lint_file(python_file)
+            elif python_file in file:
+                lint_file(python_file)
+    if tp_id == []:
```

Comment:
I don't see how this condition can be true. `tp_id` wil either be defined by the user to not the empty list, or not be defined by the user and auto-populated from `get_tp_id()`

Reply:
if someone has no tp folder and doesn't specify tp_id it could happen but i agree it shouldn't, should I still remove it ?

Reply:
Yes, I think the logic between this `if` and its previous block should be changed. Probably we don't want to glob on `*.py` if `-f` is provided.